src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/PiattaformaNotifichePGPAPage.java

```diff
@@ -121,7 +121,7 @@ public void clickNotificheDelegate() {
         try {
             getWebDriverWait(10).until(ExpectedConditions.visibilityOf(this.sideItemNotificheButton));
             sideItemNotificheButton.click();
-            By notificheDelegateButton = By.id("side-item-Notifiche delegate");
+            By notificheDelegateButton = By.id("side-item-Gaio Giulio Cesare");
```

Comment:
delete try and catch and pass name in cucumber step, not hardcoded

Reply:
Remove